test/test_nn.py

```diff
@@ -8324,6 +8323,44 @@ def eval_constructor(*args, **kwargs):
         test_params['constructor'] = gen_eval_constructor(test_params['constructor'])
         test = NewModuleTest(**test_params)
         add_test(test, decorator)
+    if 'check_with_long_tensor' in test_params:
+        fullname = test_params.get('fullname', None)
+        if fullname:
+            test_params['fullname'] = fullname + '_with_long_tensor'
+        else:
+            desc = test_params.get('desc', None)
+            test_params['desc'] = 'with_long_tensor' if desc is None else desc + '_with_long_tensor'
+
+        def gen_long_tensor(size):
+            return torch.randint(-1000, 1000, size=size).double().cpu()
+
+        def gen_long_tensor_constructor(constructor):
+            def long_tensor_constructor(*args, **kwargs):
+                cons = constructor(*args, **kwargs)
+                cons._apply(lambda t: Parameter(gen_long_tensor(t.size())) if t.is_floating_point() else t)
```

Comment:
I think it's fine to do if `t.is_floating_point and isinstance(t, Parameter)` or something.

Reply:
Done